src/libm-tester/tester.c

```diff
@@ -29,7 +29,7 @@
 #include "testerutil.h"
 
 void stop(char *mes) {
-  fprintf(stderr, "%s\n", mes);
+  printf("%s\n", mes);
```

Comment:
Why all these fprintf to printf substitution? I think you should create a separate pull request for this. You patch is big, having to review functional changes among non functional changes just slows down teh whole process. Please revert the fprintf changes for this pull request.